praw/models/reddit/subreddit.py

```diff
@@ -2093,7 +2093,7 @@ def add(self, redditor, permissions=None, **other_settings):
         user.
 
         For example, to invite ``'spez'`` with ``'posts'`` and ``'mail'``
-            permissions to ``'/r/test/``, try:
+            permissions to ``'r/test/``, try:
```

Comment:
Looks like the `'` probably shouldn't be there, nor do we need the trailing `/`.